uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Hook/NavigationMenu/ConfigurationMenuModifier.php

```diff
@@ -0,0 +1,186 @@
+    }
+  }
+
+  /**
+   * Adds a submenu containing links to edit job role option groups
+   *
+   * @param array $menu
+   */
+  private function addJobRoleItems(&$menu) {
+    $parent = 'Administer/Job Roles';
+    $permission = 'Access CiviCRM';
+
+    $optionGroupLinks = [
+      'Locations' => 'hrjc_location',
+      'Regions' => 'hrjc_region',
+      'Departments' => 'hrjc_department',
+      'Levels' => 'hrjc_level_type',
+      'Cost Centres' => 'cost_centres',
+    ];
+
+    $this->insert($menu, $parent, $permission);
+
+    foreach ($optionGroupLinks as $itemName => $optionGroup) {
+      $link = 'civicrm/admin/options/' . $optionGroup . '?reset=1';
+      $path = $parent . '/' . $itemName;
+      $this->insert($menu, $path, $permission, $link);
+    }
+  }
+
+  /**
+   * Adds a submenu containing links to edit contact related option groups,
+   * and relationship types
+   *
+   * @param array $menu
+   */
+  private function addOtherStaffDetailsMenu(&$menu) {
+    $parent = 'Administer/Other Staff Details';
+    $permission = 'Access CiviCRM';
+
+    $this->insert($menu, $parent, $permission);
+
+    $optionGroupLinks = [
+      'Prefixes' => 'individual_prefix',
+      'Emergency Contact Relationships' => 'relationship_with_employee_20150304120408',
+      'Career History' => 'occupation_type_20130617111138',
+      'Disability Types' => 'type_20130502151940',
+      'Qualifications – Skill Categories' => 'category_of_skill_20130510015438',
+      'Qualifications – Skill Levels' => 'level_of_skill_20130510015934',
+    ];
+
+    foreach ($optionGroupLinks as $itemName => $optionGroup) {
+      $link = 'civicrm/admin/options/' . $optionGroup . '?reset=1';
+      $path = $parent . '/' . $itemName;
+      $this->insert($menu, $path, $permission, $link);
+    }
+
+    // add shortcut to relationship types edit
+    $relationshipTypesLink = 'civicrm/admin/reltype?reset=1';
+    $path = $parent . '/' . 'Manager Types';
+    $this->insert($menu, $path, $permission, $relationshipTypesLink);
+  }
+
+  /**
+   * Adds links to edit option groups related to leave and absence
+   *
+   * @param array $menu
+   */
+  private function addLeaveMenuItems(&$menu) {
+    $parent = 'Administer/leave_and_absences';
+    $permission = 'administer leave and absences';
+
+    $optionGroupLinks = [
+      'Sickness Reasons' => 'hrleaveandabsences_sickness_reason',
+      'TOIL to be Accrued' => 'hrleaveandabsences_toil_amounts',
+      'Work Pattern Change Reasons' => 'hrleaveandabsences_work_pattern_change_reason',
+      'Work Pattern Day Equivalents' => 'hrleaveandabsences_leave_days_amounts'
+    ];
+
+    foreach ($optionGroupLinks as $itemName => $optionGroup) {
+      $link = 'civicrm/admin/options/' . $optionGroup . '?reset=1';
+      $path = $parent . '/' . $itemName;
+      $this->insert($menu, $path, $permission, $link);
+    }
+
+    // insert dividers
+    $numberOfDividers = 2;
+    $dividerText = '---------------------';
```

Comment:
Wow, is this really how we add a divider to a CiviCRM menu? Wouldn't it be better to add some sort of `insertDivider()` method to the NavigationMenuHelper?

Reply:
Well I don't think we use dividers anywhere else. It's a good idea to add a helper method. I think I'll maybe suggest to Mike / Jamie that we split this into a separate ticket. It's not a very elegant solution, it would probably be better to add some CSS class to nav items and apply a border to the bottom of them to create a divider. I'll remove this from the PR.